sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilderHelper.java

```diff
@@ -0,0 +1,154 @@
+    private final AzureMonitorExporterServiceVersion serviceVersion;
+
+    private final LazyConnectionString connectionString;
+    private final LazyHttpPipeline httpPipeline;
+    private final LazyTelemetryItemExporter telemetryItemExporter;
+
+    AzureMonitorExporterBuilderHelper(AzureMonitorExporterServiceVersion serviceVersion,
+                                      LazyConnectionString connectionString,
+                                      LazyTelemetryItemExporter telemetryItemExporter,
+                                      LazyHttpPipeline httpPipeline) {
+        this.serviceVersion = serviceVersion;
+        this.connectionString = connectionString;
+        this.telemetryItemExporter = telemetryItemExporter;
+        this.httpPipeline = httpPipeline;
+    }
+
+
+    /**
+     * Configures an {@link AutoConfiguredOpenTelemetrySdkBuilder} based on the options set in the builder.
+     */
+    public void build(AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder) {
+
+        sdkBuilder.addPropertiesSupplier(() -> {
+            Map<String, String> props = new HashMap<>();
+            props.put("otel.traces.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+            props.put("otel.metrics.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+            props.put("otel.logs.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+            props.put(AzureMonitorExporterProviderKeys.INTERNAL_USING_BUILDER, "true");
+            return props;
+        });
+        sdkBuilder.addSpanExporterCustomizer(
+            (spanExporter, configProperties) -> {
+                if (spanExporter instanceof AzureMonitorSpanExporterProvider.MarkerSpanExporter) {
+                    spanExporter = buildTraceExporter(configProperties);
+                }
+                return spanExporter;
+            });
+        sdkBuilder.addMetricExporterCustomizer(
+            (metricExporter, configProperties) -> {
+                if (metricExporter instanceof AzureMonitorMetricExporterProvider.MarkerMetricExporter) {
+                    metricExporter = buildMetricExporter(configProperties);
+                }
+                return metricExporter;
+            });
+        sdkBuilder.addLogRecordExporterCustomizer(
+            (logRecordExporter, configProperties) -> {
+                if (logRecordExporter instanceof AzureMonitorLogRecordExporterProvider.MarkerLogRecordExporter) {
+                    logRecordExporter = buildLogRecordExporter(configProperties);
+                }
+                return logRecordExporter;
+            });
+        // TODO
+//        sdkBuilder.addTracerProviderCustomizer((sdkTracerProviderBuilder, configProperties) -> {
+//            QuickPulse quickPulse = QuickPulse.create(getHttpPipeline());
+//            return sdkTracerProviderBuilder.addSpanProcessor(
+//                new LiveMetricsSpanProcessor(quickPulse, createSpanDataMapper()));
+//        });
```

Comment:
is this intended to be commented out for now?

Reply:
ya, I thought it would be super easy, but wasn't, so decided to leave it stubbed in here